demo/index.html

```diff
@@ -1,6 +1,7 @@
 <!doctype html>
```

Comment:
The demos are pretty short. It could be simpler if all of them were in `index.html`.